apps/web/src/routes/AuthLayoutRouter/components/ContextMenu/MenuLink/MenuLink.jsx

```diff
@@ -22,14 +36,28 @@ export default function MenuLink ({ badgeCount = null, to, children, onClick, ex
       : `https://${externalLink}`
 
     return (
-      <a href={url} target='_blank' rel='noreferrer' onClick={onClick} className={cn('MenuLink text-foreground text-sm', className, { 'opacity-100 border-selected': isCurrentLocation })}>
+      <a
+        href={url}
+        target='_blank'
+        rel='noreferrer'
+        onClick={onClick}
+        ref={isCurrentLocation ? linkRef : null}
+        className={cn('MenuLink text-foreground text-sm', className,
+          { 'opacity-100 border-selected bg-background': isCurrentLocation })}
+      >
         {children}
       </a>
     )
   }
 
   return (
-    <Link to={to} onClick={handleClick} className={cn('text-foreground text-sm focus:text-foreground relative', className, { 'opacity-100 border-selected': isCurrentLocation })}>
+    <Link
+      to={to}
+      onClick={handleClick}
+      ref={isCurrentLocation ? linkRef : null}
```

Comment:
but we should not be adding the same ref to multiple elements!

Reply:
I will clean up or revert and start over. Sorry about that! I think the use cases I was trying to fix should be accounted for -- the behavior was too confusing. Happy to roll back and let you take a stab at fixing those issues, @KevinTriplett  -- the other style improvements I made I can copy over

Reply:
Or I can fix it tomorrow :)